rust/frontend/src/lib.rs

```diff
@@ -48,15 +48,14 @@ use std::{
 use failure::Error;
 
 pub use crate::{
-    bytearray::TVMByteArray,
     context::{TVMContext, TVMDeviceType},
     errors::*,
     function::Function,
     module::Module,
     ndarray::NDArray,
     tvm_common::{
         errors as common_errors,
-        ffi::{self, TVMType},
+        ffi::{self, TVMByteArray, TVMType},
```

Comment:
`TVMByteArray` should be a re-export from common

Reply:
Isn't it already?

Reply:
top-level. FFI shouldn't be something that other crates import. `runtime` and `frontend` are still low level, but importing from `ffi` is a good indication that it should go in common proper.

Reply:
You've exported `TVMType` like that. If I want to get `TVMByteArray` from ffi, I need to wrap it like before.

Reply:
Okay, gotcha. Just making sure that it's only being used for FFI and not being leaked outside of `frontend` :)